sql/core/src/test/resources/sql-tests/results/explain.sql.out

```diff
@@ -654,6 +740,11 @@ struct<plan:string>
 
 (1) Scan parquet default.explain_temp1 
 Output: [key#x, val#x]
+Batched: true
+Format: Parquet
+Location [not included in comparison]sql/core/spark-warehouse/[not included in comparison]
+PushedFilters: [IsNotNull(key), GreaterThan(key,10)]
+ReadSchema: struct<key:int,val:int>
```

Comment:
Can we have a test case to show the DPP filter?

Reply:
@cloud-fan I have enhanced the test in ExplainSuite.